src/pages/projects/ProjectsTable/EditedAt/EditedAt.tsx

```diff
@@ -3,35 +3,31 @@ import { Button, IconKebab, NavigationList, Popover, Text } from '@gpn-prototype
 
 import { MenuItem } from '../types';
 
-import './EditedAt.css';
-
-const blockName = 'ProjectsTable-EditedAt';
-const styles = {
-  root: `${blockName}__root`,
-  editedTime: `${blockName}__editedTime`,
-  menu: `${blockName}__menu`,
-  navigation: `${blockName}__navigation`,
-  navigationItem: `${blockName}__navigationItem`,
-  iconWrap: `${blockName}__iconWrap`,
-};
+import { cnEditedAt } from './cn-edited-at';
 
 type EditedAtProps = {
   date?: string | React.ReactElement;
   menu?: MenuItem[];
   isVisible: boolean;
-  onClickItem?: VoidFunction;
+  onMenuToggle(shouldSetActiveRow: boolean): void;
```

Comment:
```suggestion
  onMenuToggle(isMenuShowed: boolean): void;
```
тут бы тоже переименовать